gears/flywheel/form-and-annotations-exporter.json

```diff
@@ -0,0 +1,63 @@
+{
+  "author": "Flywheel",
+  "cite": "",
+  "command": "/flywheel/v0/run.py",
+  "config": {
+    "annotations-scope": {
+      "default": "task",
+      "description": "The scope of the exported annotations.",
+      "enum": [
+        "task",
+        "non-task",
+        "both",
+        "none"
+      ],
+      "type": "string"
+    },
+    "debug": {
+      "default": false,
+      "description": "Log debug messages",
+      "type": "boolean"
+    },
+    "form-responses-scope": {
+      "default": "task",
+      "description": "The scope of the form responses.",
+      "enum": [
+        "task",
+        "non-task",
+        "both",
+        "none"
+      ],
+      "type": "string"
+    },
+    "protocol_name": {
+      "description": "The name of the protocol to export data from.",
+      "optional": false,
+      "type": "string"
```

Comment:
I thought that one would need to be optional in the case you are exporting with the following config no?
- form-responses-scope = none
- annotations-scope = non-task

Reply:
That is a 30-minute fix... and another gear-release to handle the cases of :

- protocol_name = "", annotations_scope in [non-task, none],form-response-scope in [non-task, none]
- protocol_name = "" , annotations_scope not in [non-task, none],form-response-scope not in [non-task, none]: raise ERROR